src/main.py

```diff
@@ -1,9 +1,25 @@
 from fastapi import FastAPI
 from routers.health import health_router
 from routers.mtcars import data_output
+from docs.documentation import description
 import uvicorn
 
-app = FastAPI()
+app = FastAPI(
+    # This info goes directly into /docs
+    title="RMI Web API poc",
+    # Description of API defined in docs/documentation.py for ease of reading
+    description=description,
+    summary="This project is a proof-of-concept (POC) web API built using the FastAPI library.",
+    version="0.0.1",
```

Comment:
This is brittle, and likley to fall out of sync. Better to derive the version of the FastAPI app directly from the `pyproject.toml` (as `pyproject.toml` will be our packages main source of truth).

Reply:
You can likely achieve this by programmatically accessing the `pyproject` meta data with:
``` python
import importlib.metadata

version = importlib.metadata.version('web-api-poc')
```

See here for info: https://stackoverflow.com/questions/75100102/get-app-version-from-pyproject-toml-inside-python-code